### DISCUSSION TITLE
Recovering a hidden column in table view

### DISCUSSION THREAD

**Discussion by stephaniemar** [community/community#6761] [Projects and Issues]
I added a new custom column/field to a project(beta) in table view, I clicked hide field in the drop down menu in the column description an I cannot figure out how to get it back. I cannot add a new field with that same name as I am getting an error that says "Name has already been taken"

**Comment by stephaniemar** [comment]
It seems when I added another Issue to the project board I was able to view hidden fields and add it back.

**Reply by Sid-ah** [reply]
Here is how you can hide/show fields in Projects Beta.

![CleanShot 2021-10-27 at 17 27 03](https://user-images.githubusercontent.com/17467117/139107491-43492ac4-abfd-4e86-90a0-5d9aefff3ffe.gif)
